### ISSUE TITLE
Hide polymorphic models fields

### ISSUE THREAD

**Issue by gabriel-cardoso** [para-cms/para#29]
Say I have 
```ruby
class Image < ActiveRecord::Base
  belongs_to :imageable, polymorphic: true, inverse_of: :image
end
```

We want to hide `imageable_id` and `imageable_type` fields

**Comment by vala** [comment]
@gabriel-cardoso Have a look at https://github.com/para-cms/para/blob/master/lib/para/model_field_parsers/relations.rb#L13-L27 You should check wether the `reflection` object has a `:polymorphic` option (by inspect the reflection object you may find how to do it quickly, then take the needed measures to treat that case, taking into account nested forms

**Comment by vala** [comment]
Since we're talking about nested forms, you may be able to handle that here https://github.com/para-cms/para/blob/master/lib/para/form_builder/nested_form.rb#L5-L7 maybe by changing the `#inverse_of?` method logic ?

**Comment by gabriel-cardoso** [comment]
Fixed by e957b12c74424fc3ec29a8455b7eaf06aa33802b
